lib/bootloader_setup.pm

```diff
@@ -899,8 +899,13 @@ sub specific_bootmenu_params {
         push @params, "autoupgrade=1";
     }
 
-    if (my $agama_auto = get_var('INST_AUTO')) {
-        my $url = ($agama_auto =~ /\.libsonnet/) ? autoyast::generate_json_profile($agama_auto) : autoyast::expand_agama_profile($agama_auto);
+    # override default boot params
+    if (get_var('BOOTPARAMS')) {
+        push @params, split ' ', trim(get_var('BOOTPARAMS'));
+    }
```

Comment:
if we don't ever use this in s390x zVM perhaps it doesn't make sense to have it and just drop it, it could be that a total replacement makes not sense. Also in the way that is implemented in qemu is not like this, it would need to return and do nothing else, so perhaps we can skip it, wdyt?

Reply:
Yes, I will drop this.